terraform/modules/gsp-cluster/main.tf

```diff
@@ -37,6 +39,9 @@ module "cluster" {
   # optional
   worker_count = 2
   worker_type  = "t2.medium"
+
+  cluster_id = "${var.cluster_name}.${var.zone_name}"
+  admin_role_arns = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/admin"]
```

Comment:
I'm not sure this is a good idea. It might be better to receive this as a list variable at the module level and pass it straight down to typhoon.